back-end/utils/image_utils.py

```diff
@@ -169,3 +169,16 @@ def get_split_length(split):
     """
 
     return len(get_image_path(split, None, None))
+
+def get_classified_split_length(split):
+    """
+    Get the length of a data split
+
+    args: 
+        split:  e.g. 'train', 'validation', 'test_correct'
+
+    returns:
+        The length of the data split as an integer
+    """
+    if split in ["validation", "test"]:
```

Comment:
This works, but since returns nothing to the frontend if split is invalid, it causes the following error in the browser console

```
vue.runtime.esm.js?2b0e:1897 TypeError: Cannot read properties of null (reading '0')
    at VueComponent.accuarcy (ImageList.vue?22bb:346:1)
    at Watcher.get (vue.runtime.esm.js?2b0e:4495:1)
    at Watcher.evaluate (vue.runtime.esm.js?2b0e:4597:1)
    at VueComponent.computedGetter [as accuarcy] (vue.runtime.esm.js?2b0e:4851:1)
    at Object.get (vue.runtime.esm.js?2b0e:2081:1)
    at Proxy.render (ImageList.vue?0cd7:124:1)
    at Vue._render (vue.runtime.esm.js?2b0e:3569:1)
    at VueComponent.updateComponent (vue.runtime.esm.js?2b0e:4081:1)
    at Watcher.get (vue.runtime.esm.js?2b0e:4495:1)
    at Watcher.run (vue.runtime.esm.js?2b0e:4570:1)

... 
vue.runtime.esm.js?2b0e:619 [Vue warn]: Error in render: "TypeError: Cannot read properties of null (reading '0')"

found in

---> <ImageList> at src/views/ImageList.vue
       <VMain>
         <VApp>
           <App> at src/App.vue
             <Root>
```

To disable the error, try returning `RResponse` with something else and handle it correctly in the frontend.